modules/imgproc/src/color.cpp

```diff
@@ -9105,40 +9113,40 @@ struct RGB888toYUV420pInvoker: public ParallelLoopBody
                 int u00 = ITUR_BT_601_CRU * r00 + ITUR_BT_601_CGU * g00 + ITUR_BT_601_CBU * b00 + halfShift + shifted128;
                 int v00 = ITUR_BT_601_CBU * r00 + ITUR_BT_601_CGV * g00 + ITUR_BT_601_CBV * b00 + halfShift + shifted128;
 
-                u[k] = saturate_cast<uchar>(u00 >> ITUR_BT_601_SHIFT);
-                v[k] = saturate_cast<uchar>(v00 >> ITUR_BT_601_SHIFT);
+                if (interleaved)
+                {
+                    u[k*2] = saturate_cast<uchar>(u00 >> ITUR_BT_601_SHIFT);
+                    v[k*2] = saturate_cast<uchar>(v00 >> ITUR_BT_601_SHIFT);
+                }
+                else
+                {
+                    u[k] = saturate_cast<uchar>(u00 >> ITUR_BT_601_SHIFT);
+                    v[k] = saturate_cast<uchar>(v00 >> ITUR_BT_601_SHIFT);
+                }
             }
         }
     }
 
-    static bool isFit( int src_width, int src_height )
+    void convert() const
     {
-        return (src_width * src_height >= 320*240);
+        if( src_width * src_height >= 320*240 )
+            parallel_for_(Range(0, src_height/2), *this);
+        else
+            operator()(Range(0, src_height/2));
     }
 
 private:
     RGB888toYUV420pInvoker& operator=(const RGB888toYUV420pInvoker&);
```

Comment:
do we need hidden = operator? probably, "const int scn" can be changed to "int scn". Since `operator()(const Range& r)` is `const` method, all the class members are automatically treated as constants

Reply:
This is for static code analyzers. There is no implementation. This is just a BKM.

In C++11 this should look like this:
```
RGB888toYUV420pInvoker& operator=(const RGB888toYUV420pInvoker&) = delete;
```
The same for copy constructor.

Without this compiler would generate default operator (with invalid imlementation), which may be used by mistake.